glean-core/src/storage/mod.rs

```diff
@@ -55,4 +55,26 @@ impl StorageManager {
             Some(json!(snapshot))
         }
     }
+
+    pub fn snapshot_metric(
```

Comment:
Can you please add documentation about what does the function do, its parameters and the return value? :)

Reply:
Ha, damn, you got me.